### PR TITLE
New README format

### PR DESCRIPTION
**Pull request by autero1**
This PR updates the README to the new format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
+:type: service
+:name: TICK Stack GCP
+:description: Deploy the TICK stack (Telegraf, InfluxDB, Chronograf, Kapacitor) in GCP to gather and process time series data.
+:icon: /_docs/tick-stack-gcp-icon.png
+:category: Other data stores
+:cloud: gcp
+:tags: database, time-series
+:license: open-source
+:built-with: terraform, bash
+-->
```
Comment (autero1) [inline README.md]: Will have to double check the metadata

File: README.md [hunk 2]
```diff
@@ -1,21 +1,48 @@
+<!--
+:type: service
+:name: TICK Stack GCP
```
Comment (yorinasub17) [inline README.md]: My understanding is that the name should be just "TICK Stack", and the GCP nature of it is pulled in by the `cloud` tag. But I could be wrong: @eak12913 is that a correct assumption?

File: README.md [hunk 3]
```diff
@@ -1,21 +1,48 @@
...
+:description: Deploy the TICK stack (Telegraf, InfluxDB, Chronograf, Kapacitor) in GCP to gather and process time series data.
+:icon: /_docs/tick-stack-gcp-icon.png
+:category: Other data stores
```
Comment (yorinasub17) [inline README.md]: This should be "Monitoring & alerting"

File: README.md [hunk 4]
```diff
@@ -1,21 +1,48 @@
...
 This repo contains the **official** module for deploying the [TICK Stack](https://www.influxdata.com/time-series-platform/) on [GCP](https://cloud.google.com/gcp/) using [Terraform](https://www.terraform.io/) and [Packer](https://www.packer.io/).
 
+## TICK Stack Architecture
+
+![TICK multi-cluster architecture](https://github.com/gruntwork-io/terraform-google-influx/blob/master/_docs/tick-multi-cluster-architecture.png?raw=true)
+
+## Features
+
+- Deploy the TICK stack (Telegraf, InfluxDB, Chronograf, Kapacitor) to gather and process time series data.
+- Supports both InfluxDB Enterprise and OSS
```
Comment (yorinasub17) [inline README.md]: I also had the following when I did nomad (https://github.com/hashicorp/terraform-aws-nomad), which I think also applies here:

- Supports colocated clusters and separate clusters
- Auto scaling and Auto healing

File: README.md (hunk 2 above)
Comment (eak12913) [reply README.md]: @yorinasub17 - that's correct.

Comment (autero1) [reply README.md]: @eak12913 is `:name: TICK Stack` still valid for the `README-InfluxDB.md`, or should it be `InfluxDB`?

**Comment by autero1** [comment]
Thanks for the review - merging this in

**Pull request merged**